photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -514,18 +551,22 @@ final class Driver(val params: Params, val sparkContext: SparkContext, val logge
 
     trainingDataSet.foreach { case (_, rddLike: RDDLike) => rddLike.unpersistRDD() }
 
+    timer.start()
+    val bestModel = selectBestModel(gameModelsMap, validatingDataAndEvaluatorsOption)
+    timer.stop()
+    logger.info(s"Time elapsed after selecting best model: ${timer.durationSeconds} (s)\n")
+
```

Comment:
Fair enough, in that case we should move the output to log of the best model from `saveModelToHDFS` to `selectBestModel`.